zapisy/apps/theses/models.py

```diff
@@ -125,6 +126,12 @@ def is_supporting_advisor_assigned(self, user):
     def is_among_advisors(self, user):
         return self.is_mine(user) or self.is_supporting_advisor_assigned(user)
 
+    def is_user_privileged(self, user):
+        return self.is_among_advisors(user) or user.is_staff or is_theses_board_member(user)
+
+    def can_user_generate_declarations(self, user):
```

Comment:
```suggestion
    def can_user_download_declarations(self, user):
```
Raz używamy jednego, raz drugiego czasownika, i o ile ja bym nieznacznie wolał `generate`, to łatwiej się go pozbyć z tej nazwy (i odwołań do niej).